OpenNefia.Core/Locale/LocalizationManager.cs

```diff
@@ -216,5 +223,47 @@ public bool TryGetLocalizationData(EntityUid uid, [NotNullWhen(true)] out LuaTab
 
             return TryGetTable($"OpenNefia.Prototypes.Entity.{metadata.EntityPrototype.ID}", out table);
         }
+
+        private void MungeEcksdee()
```

Comment:
Rename to `CacheEntityLocData()`.

Reply:
opted to remove instead, is changed now.